pkg/controller/pod.go

```diff
@@ -618,13 +619,14 @@ func (c *Controller) handleUpdatePodSecurity(key string) error {
 		utilruntime.HandleError(fmt.Errorf("invalid resource key: %s", key))
 		return nil
 	}
-	pod, err := c.podsLister.Pods(namespace).Get(name)
+	oripod, err := c.podsLister.Pods(namespace).Get(name)
```

Comment:
the variable pod here is not modified in context

Reply:
modified